persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -122,6 +126,17 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor {
     case GetState =>
       sender() ! state
 
+    case GetStatePaginated(page) =>
+      val tsErrState = state.tsParsingErrors
+      val tsErrorCount = if (tsErrState.isEmpty) 0 else 1
+      val tsErrorsReturn = if (tsErrState.nonEmpty && page == 1) tsErrState else Seq()
```

Comment:
Oops, that was left over from a previous strategy. Good catch. 

Fixed in dc0a6fce17c1f6d467f206d661e7f16af7a517b0.